modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -152,6 +164,44 @@ sda.enkf.multisite <- function(settings,
     site.ids <- as.character(settings$run$site$id)
   }
   
+  # Build site coords once (used by covariate extraction)
+  site_coords <- purrr::map_df(settings$run, ~ tibble::tibble(
```

Comment:
We can avoid needing to add another package to our (already long) import list by using `dplyr::tibble` -- it's the same function, just re-exported by dplyr for exactly this reason.
```suggestion
  site_coords <- purrr::map_df(settings$run, ~ dplyr::tibble(
```